### ISSUE TITLE
OBJECT DETECTION FOR SMALL OBJECT

### ISSUE THREAD

**Issue by masgh021** [dotnet/machinelearning#5844]
OBJECT DETECTION FOR SMALL OBJECT
RETURN WRONG VALUE . 
WHY ?

**Comment by michaelgsharp** [comment]
Can you share more info? Either a code sample or a sample project?

**Comment by masgh021** [comment]
my image is license plate 
and i want to recognize digits from it .
but it sends wrong value , 
digit size are 25px * 30px .
but result is not correct .

**Comment by LittleLittleCloud** [comment]
Did you resize the bounding boxes to fit your image size?
See the following issue for more details.
https://github.com/dotnet/machinelearning/issues/5772

**Comment by masgh021** [comment]
@LittleLittleCloud 
@michaelgsharp 
yes i did , but it does not recognize any  tiny(small) object  like digits in plate or characters in text .

**Comment by LittleLittleCloud** [comment]
Hi @masgh021 
Sounds like your task should be more of an OCR rather than object detection? The difference is

- OCR -> find digit in plate
- object detection -> find plate

Can you share a sample picture how large the digit number is? Object detection might work if you crop around the plate and label each digit as an object? But OCR will perform better.

**Comment by masgh021** [comment]
like this link ,
[image](http://www.licenseplatemania.com/fotos/rusland/rusland22.jpg) 
i  found plate ,
but i can't find digit number with object detection .

**Comment by LittleLittleCloud** [comment]
Yeah this is something that object detection might not work well with. I would try OCR instead.

**Comment by masgh021** [comment]
which OCR do you use  for c# ?(opensource)
i want use for custom digit and character , it has good accuracy .
i can train it easily .

**Comment by briacht** [comment]
@masgh021 I know a lot of people have success with OpenCV. This article may help you: https://www.tech-quantum.com/ocr-text-using-opencv-and-tesseract/

Also here's an example of a customer who uses ML.NET to detect license plates, and then they use separate technology for the OCR part: https://dotnet.microsoft.com/apps/machinelearning-ai/ml-dotnet/customers/scancam

**Comment by rosebyte** [comment]
Object detection can locate the licence plate, but recognising its individual characters requires OCR. The recommended approach has been provided above, and there is no actionable ML.NET defect here, so we are closing this issue.
